doc/man3/PKCS12_create.pod

```diff
@@ -32,13 +32,10 @@ B<keytype> is the type of key.
 The parameters B<nid_key>, B<nid_cert>, B<iter>, B<mac_iter> and B<keytype>
 can all be set to zero and sensible defaults will be used.
 
-These defaults are: 40 bit RC2 encryption for certificates, triple DES
-encryption for private keys, a key iteration count of PKCS12_DEFAULT_ITER
-(currently 2048) and a MAC iteration count of 1.
-
-The default MAC iteration count is 1 in order to retain compatibility with
-old software which did not interpret MAC iteration counts. If such compatibility
-is not required then B<mac_iter> should be set to PKCS12_DEFAULT_ITER.
+These defaults are: AES password based encryption (PBES2 with PBKDF2 and
+AES-256-CBC) for private keys and certificates, the PBKDF2 and MAC key iteration
```

Comment:
MAC key iteration -> MAC iteration

Reply:
But it is MAC KDF iteration count not an iteration of the MAC itself.

Reply:
Hmm, are you sure?
At least in the doc this was stated differently so far,
and the value is passed to `PKCS12_set_mac()` and then `PKCS12_setup_mac()`,
both of which unfortunately are undocumented, but they appear to set the mac iteration count itself.

Reply:
There is no such thing as mac iteration count really. We just call the MAC key derivation iteration count sloppily in short mac iteration count, however that is misleading as it looks like we would do the whole mac computation many times.

Reply:
Please look at the implemenation in p12_mutl.c if you do not believe me. :grin: